app/models/work.rb

```diff
@@ -0,0 +1,43 @@
+class Work < ApplicationRecord
+  validates :title, presence: true
+  validates_uniqueness_of :title, scope: :category
+
+  has_many :votes, dependent: :destroy
+
+  def self.sort_works(category)
```

Comment:
Suggestion, make the valid categories a constant array and then you can check to see if the category is valid with CATEGORIES.include? category

Then you can use the variable with `self.where(category: category)`